siriuspy/tests/currinfo/test_main.py

```diff
@@ -0,0 +1,55 @@
+# #!/usr/bin/env python-sirius
```

Comment:
Is it really worthy to keep a whole file commented out like this? With git tools it is easy to retrieve code whenever we want...
Couldn't we remove this file?

Reply:
I tried to fix these tests instead of commenting them, but I think I will remove them momentarily. In another opportunity I fix all new tests that I brought from machine-applications.